Makefile

```diff
@@ -149,7 +151,7 @@ clean:
 	echo "The Emsdk, CPython and CLAPACK are not cleaned. cd into those directories to do so."
 
 
-%.bc: %.c $(CPYTHONLIB) $(LZ4LIB)
+%.bc: %.c $(CPYTHONLIB) $(LZ4LIB) $(LIBXML) $(LIBXSLT)
```

Comment:
Is this needed as a dependency of these?  These `.c` files don't depend on libxml or libxslt.  (But your package does, so it makes sense as a dependency of the packages build below...)

Reply:
You are 100% right :)